srcpkgs/xwinwrap/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'xwinwrap'
+pkgname=xwinwrap
+version=0.9
+revision=1
+build_style=gnu-makefile
+makedepends="libXrender-devel"
+short_desc="X11 Window as background"
+maintainer="SolitudeSF <solitudesf@protonmail.com>"
+license="GPL-3.0-or-later"
```

Comment:
it's MIT-like licensed

Reply:
how should i specify that?

Reply:
License text is in source code file. `head` it into `LICENSE` file, include into package with `vlicense` and use similar enough identifier of SPDX.